lib/init/rc.shutdown

```diff
@@ -8,8 +8,10 @@ main() {
     PATH=/sbin:/bin:/usr/sbin:/usr/bin
 
     log "Waiting for services to stop..."; {
-        sv -w196 force-stop /var/service/*
-        sv exit /var/service/*
+        if command -v sv && [ "$(pgrep runsvdir)" ]; then
```

Comment:
`command -v sv && pgrep runsvdir >/dev/null 2>&1 && {`

Then replace `fi` with `}`.

(Also, see my IRC messages)

Reply:
An alternative which is simpler is to just add `>/dev/null 2>&1` to both `sv` calls. If the command isn't installed nothing happens and if it is installed but not in use, nothing will happen also.

Reply:
Simpler can be better.